modules/lvm2/udiskslvm2moduleiface.c

```diff
@@ -154,7 +154,6 @@ lvm_update_from_variant (GPid      pid,
           udisks_linux_volume_group_object_destroy (group);
           g_dbus_object_manager_server_unexport (manager,
                                                  g_dbus_object_get_object_path (G_DBUS_OBJECT (group)));
-          g_object_unref (G_OBJECT (group));
```

Comment:
@vpodzime Even though we fork & exec a separate process to retrieve the information about each VG, the results processing call back always appears to be handled in the main thread.  So it appears that we don't need to protect the hash tables from concurrent access for lvm plugin as there is only one thread of execution mucking with them.